internetarchive/openlibrary
# chg: usr: Remove OCAID from UI for book add/edit if user is not admin…

… books/edit for users who are not admin or librarian.


Closes #5135 



Fix issue where non-administrative user was presented with option to add or edit OCAID.

### Technical


Changes were made to the templates supporting functionality which allows users to add and edit books.


### Testing

* Log in as administrative user, navigate to `/books/add`.
* Confirm that "Internet Archive" exists in list for optional ID number.
* Fill out the form, add the book.
* Edit book, confirm that optional ID includes "Internet Archive"
* Log in as non-administrative user, navigate to `/books/add`.
* Confirm that "Internet Archive" does not exist in optional ID list
* Fill out the form, add the book.
* Edit the book, confirm that "Internet Archive" does not exist in optional ID list.

### Screenshot

Admin creates a book:
<img width="825" alt="AdminCreate" src="https://user-images.githubusercontent.com/67606267/124878278-c642b300-df80-11eb-9c54-ba321fc5ec4c.png">

Admin edits a book:
<img width="840" alt="AdminEdit" src="https://user-images.githubusercontent.com/67606267/124878300-cb9ffd80-df80-11eb-9cd3-9742f6dbcf1a.png">

Non-administrative user creates a book:
<img width="683" alt="UserCreate" src="https://user-images.githubusercontent.com/67606267/124878314-d064b180-df80-11eb-96cc-7406c5fc4525.png">

Non-administrative user edits a book:
<img width="684" alt="UserEdit" src="https://user-images.githubusercontent.com/67606267/124878331-d5296580-df80-11eb-9340-470d33d63b9f.png">

### Stakeholders
@mekarpeles @cdrini @seabelis
